package.json

```diff
@@ -17,7 +17,7 @@
 		"@guardian/ab-core": "^4.0.0",
 		"@guardian/atom-renderer": "1.2.2",
 		"@guardian/automat-modules": "^0.3.8",
-		"@guardian/braze-components": "^11.0.0",
+		"@guardian/braze-components": "^10.0.1-0",
```

Comment:
Just a reminder that I think we said after merging the braze-components PR we'd do an v11.1.0 release so we'd update to that version here.

Reply:
yes